### PR TITLE
fix #11

### LINKED ISSUE #11: cd to subdirectory of ~ does not work

### PR DESCRIPTION
**Pull request by sadanslargehole**
needs a bit of work
also needed to change the branch + accidentally closed the first one

### CODE REVIEW

**Comment by sadanslargehole** [comment]
side effect: add support for cd without any args to take you home

**Review by keli5 — APPROVED** [review]
Looks good! Closes #11.

**Comment by keli5** [comment]
just royally fucked up the git beyond anything you can ever even hope to imagine but it is what it is, this code is upstream now
